contracts/UpkeepRegistrationRequests.sol

```diff
@@ -134,23 +146,42 @@ contract UpkeepRegistrationRequests is Owned {
         uint32 gasLimit,
         address adminAddress,
         bytes calldata checkData,
-        uint96 amount,
         bytes32 hash
     )
       external
       onlyOwner()
     {
+        PendingRequest memory request = s_pendingRequests[hash];
+        require(request.admin != address(0), "request not found");
```

Comment:
We should also have a requirement in `register` that the address is not zero. A user could lock funds in an unrecoverable way otherwise.